docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Confiuguration options for dagster_cloud.yaml | Dagster Docs
+description: How to use dagster_cloud.yaml to configure Dagster Cloud
+---
+
+# Dagster_cloud.yaml Guide
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+This dagster_cloud.yaml file is used to configure how a code location should be build and deployed to Dagster Cloud, usually as part of CI/CD. The file is required to tell Dagster where the code sources are to load into Dagster Cloud but has other options as well. For Hybrid Deployments, secrets are configured in this file as well since those secrets are part of the code location.
+
+## Where to put the dagster_cloud.yaml file
+
+The dagster_cloud.yaml file should be placed in your dagster project's repository root. Below is an example of a file structure from the [Dagster cloud ETL quickstart](https://github.com/dagster-io/dagster/tree/master/examples/quickstart_etl).
+
+```shell
+quickstart_etl
+├── README.md
+├── quickstart_etl
+│   ├── __init__.py
+│   ├── assets
+├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+
+## Code Locations
+
+The only **mandatory** section in the dagster_cloud.yaml file is setting up your code locations. If you have multiple code locations, you will have multiple code locations within a single dagster_cloud.yaml file. Dagster code can be sourced from a package, a python file, or a module, and one of these parameters should be specified for each code location.
```

Comment:
I think we are describing both the top-level `locations` key and the `code_source` key within each location here in a way that might be confusing.

Consider something like: "The only **mandatory** section in the dagster_cloud.yaml file is the `locations` key that specifies a list of code locations that should be deployed to Dagster Cloud. Each item in the list of locations must also include a `location_name` key that specifies the name of a location and a `code_source` key that specifies where to find the definitions in the code location.

Realizing i just basically replicated shalabh's comment below